gravitee-apim-console-webui/src/entities/management-api-v2/portalNavigationItem/portalNavigationItem.ts

```diff
@@ -58,3 +58,24 @@ export interface NewLinkPortalNavigationItem extends BaseNewPortalNavigationItem
 }
 
 export type NewPortalNavigationItem = NewPagePortalNavigationItem | NewFolderPortalNavigationItem | NewLinkPortalNavigationItem;
+
+interface BaseUpdatePortalNavigationItem<T extends PortalNavigationItemType> {
+  id: string;
+  title: string;
+  type: T;
+  area: PortalArea;
```

Comment:
`id` and `area` should not be updatable
```suggestion
  title: string;
  type: T;